openapi.yml

```diff
@@ -1556,6 +1506,78 @@ components:
         - label
         - type
         - version
+    RequestDescription:
```

Comment:
this could have a description itself, like "same as Description but without purl property" or "Description to be sent as part of a Request object" or something.